web/public/js/directives/cd-action-bar/style.less

```diff
@@ -0,0 +1,88 @@
+.cd-action-bar {
+
+  @keyframes bounceInUp {
+    from, 30%, 60%, 70%, 90%, to {
+      animation-timing-function: cubic-bezier(0.215, 0.610, 0.355, 1.000);
+    }
+
+    from {
+      opacity: 0;
+      transform: translate3d(0, 100%, 0);
+    }
+
+    30% {
+      opacity: 1;
+      transform: translate3d(0, -15px, 0);
+    }
+
+    45% {
+      transform: translate3d(0, 10px, 0);
+    }
+
+    60% {
+      transform: translate3d(0, -5px, 0);
+    }
+
+    70% {
+      transform: translate3d(0, 0, 0);
+    }
+
+    to {
+      transform: translate3d(0, 0, 0);
+    }
+  }
+
+  @keyframes slideOutDown {
+    from {
+      transform: translateY(0);
+    }
+
+    to {
+      transform: translateY(100%);
+    }
+  }
+
+  background: @cd-blue;
+  height: 40px;
+  z-index: 5000;
+  max-width: 1280px;
+  margin: 0 auto;
+  display: flex;
+  transform: translateY(100%);
+  animation-fill-mode: both;
+  padding-bottom: 40px;
+  margin-bottom: -40px;
```

Comment:
Similar hack, different reason. This is just so when the action bar bounces up that there's no white gap underneath it. For this, it's either fixed to the bottom, so a minus bottom margin going off the page, or is underneath the footer thanks to z-index and therefore doesn't affect click-ability.